components/admin_console/system_user_detail/system_user_detail.jsx

```diff
@@ -180,10 +262,12 @@ export default class SystemUserDetail extends React.Component {
             />
         );
 
-        if (user) {
-            firstName = user.first_name ? user.first_name : '(First name)';
-            lastName = user.last_name ? user.last_name : '(Last name)';
-            nickname = user.nickname ? user.nickname : '(Nickname)';
+        if (user.id) {
+            firstLastNickname = user.first_name + ' ' + user.last_name + ' • ' + user.nickname;
+            position = user.position ? user.position : '';
+            if (!user.nickname) {
```

Comment:
Maybe group this with the line that first sets `firstLastNickname` so that the logic is all in one place? An `if/else` would also make it really clear that you either get the full name or the full name plus nickname